openpype/modules/sync_server/sync_server.py

```diff
@@ -1,12 +1,23 @@
 """Python 3 only implementation."""
 import os
+import shutil
 import asyncio
 import threading
 import concurrent.futures
+from time import sleep
 from concurrent.futures._base import CancelledError
 
 from .providers import lib
+from openpype.client.entity_links import get_linked_representation_id
 from openpype.lib import Logger
+from openpype.lib.local_settings import get_local_site_id
+from openpype.modules.base import ModulesManager
+from openpype.pipeline import Anatomy
```

Comment:
This is public definition of sync server which must be at this moment Python 2 compatible and should be clear without pipeline imports if possible.

BTW Even I understand the use case I don't think those functions should be called directly from blender as blender workflow is now dependent on sync server in it's core (which is not good). He should find if there is sync server addon, if it's enabled, and then call a method which does all the logic and return or raise result, but blender should not know anything about how it works (whole content of `DownloadLastWorkfile`).

Reply:
Okay, I see what you mean. I'll edit my implementation.